app/presenters/idv/by_mail/letter_requested_email_presenter.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Idv
+  module ByMail
+    class LetterRequestedEmailPresenter
+      include ActionView::Helpers::TranslationHelper
+      include Rails.application.routes.url_helpers
+
+      attr_reader :current_user, :service_provider, :url_options
+      def initialize(current_user:, url_options:)
+        @current_user = current_user
+        @url_options = url_options
+        @service_provider = current_user.pending_profile&.initiating_service_provider
+      end
+
+      def sp_name
+        service_provider&.friendly_name
+      end
+
+      def show_sp_contact_instructions?
+        sp_name.present?
+      end
+
+      def show_cta?
```

Comment:
Author's note: Changed the name from `show_sign_in_cta?` to just `show_cta?` to bring it inline with the language used in other presenters.